sdk/tests/conformance2/rendering/00_test_list.txt

```diff
@@ -1,4 +1,5 @@
 attrib-type-match.html
+--min-version 2.0.1 blitframebuffer-filter-outofbounds.html
```

Comment:
Sorry, again, can we remove this min-version?  This should be part of 2.0.0 in my opinion.